storage-snippets/list_objects_in_bucket.py

```diff
@@ -6,6 +6,6 @@
 my_bucket = os.getenv('WORKSPACE_BUCKET')
 
 # List objects in the bucket
-os.system(f"gsutil ls '{my_bucket}'")
+print(subprocess.check_output(f"gsutil ls {my_bucket}", shell=True).decode('utf-8'))
```

Comment:
It's not Python, but a much simpler way of doing this would be to just do a `!gsutil ls $WORKSPACE_BUCKET`. I'm not sure this is offering much value on top of that, since I wouldn't really be able to programmatically interact with the return value here anyways.

To fix the tests, you could add some code here which just comments out or omits lines which start with `!` before writing to the smoke test file: https://github.com/all-of-us/workbench-snippets/blob/master/build/generate_jupyter_snippets_menu_extension_config.py#L67

Up to you whether it's worth the effort.